docs/_getting_started/jupyter.md

```diff
@@ -133,7 +146,9 @@ updateDate: 2023-12-18
 ---
 -->
 
-# Custom JupyterHub Python Kernel
+
+## Custom JupyterHub Python Kernel
+------------
 To make an active `venv` into a kernel in your JupyterHub, make sure you have an active `venv`:
 
 ```bash
```

Comment:
Use best practices for input and output -- https://github.com/gsbdarc/rcpedia_mockup/blob/alex_metadata/markdown_guide.md

Reply:
done